views/offramp/offramp.pug

```diff
@@ -0,0 +1,17 @@
+extends ../base
+include ../_includes/banner
+
+block variables
+  -var title = 'Your taxes can’t be completed using this service'
+
+block content
+
+  h1 #{title}
+  +blueBanner('Your tax return is too complex to be filed using this service. Nothing about your taxes is incorrect – see below for further instructions on how to complete your tax return.', 'blue')
+  div
+    p To finish filing your taxes for the current tax year (filing for 2018), you will need to do one of the following:
+    ul
+      li Use the paper form enclosed in the Claim Tax Benefits package you received in the mail
+      li Contact CRA at (123) 456-7890 for help
+      li File your taxes with another free service, listed here: #[a(href="#") Ways to file your taxes]
```

Comment:
A few comments about this:

- Let's add a new link: "[Visit a community tax clinic near you](https://www.canada.ca/en/revenue-agency/services/tax/individuals/community-volunteer-income-tax-program.html) to have a volunteer help you submit your taxes."
- Let's make the phone number a "tel" link: https://css-tricks.com/the-current-state-of-telephone-links/
- For the last point, let's say: "File your taxes with another [NETFILE-compliant online service certified by the CRA](https://www.canada.ca/en/revenue-agency/services/e-services/e-services-individuals/netfile-overview/certified-software-netfile-program.html)"